src/MagentoHackathon/Composer/Magento/Plugin.php

```diff
@@ -311,7 +311,12 @@ private function saveVendorDirPath(Composer $composer)
             realpath($composer->getConfig()->get('vendor-dir')),
             true
         );
-        $vendorPathFile = $magentoDir . '/app/etc/vendor_path.php';
+        $etcDir = $magentoDir . '/app/etc';
+        if (file_exists($etcDir)) {
+            $vendorPathFile = $etcDir . '/vendor_path.php';
+        } else {
+            $vendorPathFile = $magentoDir . '/vendor_path.php';
```

Comment:
Why is this change needed? What use case does it cover?

Reply:
This is needed when composer installer is used for functional acceptance test modules, where `app/code` is not available.